src/components/ScheduleGenerate/ScheduleGenerateModal.vue

```diff
@@ -27,25 +27,17 @@
           </div>
         </div>
         <div class="schedule-generate-body">
+          <div class="schedule-generate-subHeader schedule-generate-subHeader--smallerIndent">
+            Your Courses
+          </div>
           <div class="schedule-generate-section-courses">
-            <div class="schedule-generate-subHeader schedule-generate-subHeader--smallerIndent">
-              <span class="schedule-generate-subHeader--font">Your Courses</span>
-            </div>
-            <div class="schedule-generate-inputs">
-              <div class="schedule-generate-inputWrapper">
-                <schedule-courses :num-credits="12" :classes="classes" />
-              </div>
-            </div>
+            <schedule-courses :num-credits="12" :classes="classes" />
           </div>
           <div class="schedule-generate-section-schedule">
             <div class="schedule-generate-subHeader schedule-generate-subHeader--indent">
-              <span class="schedule-generate-subHeader--font">Fall 2024</span>
-            </div>
-            <div class="schedule-generate-inputs">
-              <div class="schedule-generate-inputWrapper">
-                <schedule :classesSchedule="classesSchedule" />
-              </div>
+              Fall 2024
```

Comment:
This should probably be a prop (selectedSemester), like you have for `ScheduleGenerateSideBar.vue` that you pass in from `Dashboard.vue`.